fdmi/ut/sd_release_fom.c

```diff
@@ -197,6 +197,8 @@ void fdmi_sd_release_fom(void)
 	m0_fdmi_source_deregister(src);
 	m0_fdmi_source_free(src);
 	m0_semaphore_fini(&g_sem);
+	/* At this moment, the rr fom may be still running. Wait a bit. */
+	m0_nanosleep(m0_time(2, 0), NULL);
```

Comment:
Can this sleep be avoided ?

Reply:
This sleep() is to avoid another issue. (not the one we mentioned).

The "record release" fop is imitated in imitate_release_fop_recv() on Line 178.
But, the "RR" (record release) fop/fom is going to be finalized.
That is running in parallel with the ut thread.
We don't have a semaphore for that at this moment.

This is UT code. No harm to wait 2 seconds for the rr fom to complete.

Reply:
Yes, this is fixed in f040fe1